src/app/shared/exception.handler.ts

```diff
@@ -32,8 +32,42 @@ export class RavenExceptionHandler extends ErrorHandler {
         id: userService.currentLoggedInUser.id
       });
     }
+
     if (environment === 'prod-preview' || environment === 'production') {
-      Raven.captureException(err.originalException || err);
+      let ex = err.originalException || err;
```

Comment:
Can the fingerprint creation be moved to a separate function; it is more intuitive to modify later.